packages/babel-plugin-polyfill-es-shims/src/index.ts

```diff
@@ -30,8 +28,8 @@ export default defineProvider<{||}>(function({
   });
 
   function createDescIterator(
-    cb: (Descriptor, Utils, Object) => void,
-    instance?: (MetaDescriptor, *, Utils, Object) => void,
+    cb: (c: Descriptor, b: Utils, a: any) => void,
+    instance?: (d: MetaDescriptor, c: any, b: Utils, a: any) => void,
```

Comment:
this is a `flowts` thing -- let me know if there's any specific var names you'd prefer instead

Reply:
```suggestion
    cb: (desc: Descriptor, utils: Utils, path: NodePath) => void,
    instance?: (meta: MetaDescriptor, resolved: any, utils: Utils, path: NodePath) => void,
```

Reply:
I renamed everything but left out the `NodePath` refinement for now

Reply:
See [follow-up discussion](https://github.com/babel/babel-polyfills/pull/121#discussion_r840848747)